src/components/scenes/EditTokenScene.tsx

```diff
@@ -98,13 +98,37 @@ function EditTokenSceneComponent(props: Props) {
     }
 
     if (tokenId != null) {
-      await wallet.currencyConfig.changeCustomToken(tokenId, token)
+      await wallet.currencyConfig.changeCustomToken(tokenId, customTokenInput)
+      navigation.goBack()
     } else {
-      const tokenId = await wallet.currencyConfig.addCustomToken(token)
-      await wallet.changeEnabledTokenIds([...wallet.enabledTokenIds, tokenId])
-      logActivity(`Add Custom Token: ${account.username} -- ${getWalletName(wallet)} -- ${wallet.type} -- ${tokenId} -- ${currencyCode} -- ${decimals}`)
+      const { currencyConfig } = wallet
+      const builtinTokens = Object.values(currencyConfig.builtinTokens)
+
+      // Check if custom token input conflicts with built-in tokens
+      const matchingContractToken = builtinTokens.find(
+        builtInToken => builtInToken.networkLocation?.contractAddress === customTokenInput.networkLocation?.contractAddress
+      )
+
+      const warningMessage = builtinTokens.find(builtInToken => builtInToken.currencyCode === currencyCode)
```

Comment:
This line will not pass the eslint strict boolean rule. Please add `!= null`

Reply:
I didn't get any violations on yarn lint